src/api/app/controllers/webui/request_controller.rb

```diff
@@ -15,7 +15,12 @@ def add_reviewer_dialog
   end
 
   def add_reviewer
-    required_parameters :review_type
+    unless params[:review_type]
+      flash[:error] = 'Type required.'
```

Comment:
you have down another error message without `.`. Use a consistent way... check if we are using `.` in other parts of the projects for example. :wink:

Reply:
Done. :poop: :poop: